CSS/style.css

```diff
@@ -0,0 +1,271 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+div,
+p {
+  line-height: 15px;
+}
+
+/* Header starts */
+header {
+  /* width: 1300px;
+  margin: 0 auto; */
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))